applications/DEMApplication/custom_constitutive/DEM_D_DMT_cohesive_law.cpp

```diff
@@ -32,17 +32,17 @@ namespace Kratos {
         const double radius_sum     = my_radius + other_radius;
         const double radius_sum_inv = 1.0 / radius_sum;
         const double equiv_radius   = my_radius * other_radius * radius_sum_inv;
-        const double cohesive_force = 8.0 * Globals::Pi * equiv_cohesion * equiv_radius;
+        const double cohesive_force = 2.0 * Globals::Pi * equiv_cohesion * equiv_radius;
```

Comment:
@salvalatorre : you coded this. Do you remember why 8.0? 
@joaquinirazabal : any clarification is welcome.

Reply:
I think that 8.0 came from the JKR_CL but in the DMT_CL it should be 2.0,